linkstone-annotations/src/main/java/net/glowstone/linkstone/annotations/LOverride.java

```diff
@@ -0,0 +1,15 @@
+package net.glowstone.linkstone.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This annotation states explicitly that the annotated method overrides a method with
+ * a {@link LMethod} annotation.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface LOverride {
```

Comment:
Can `@LImplements` be merged into this to avoid confusion?

Reply:
`@LOverride` indicates that the method exists in NMS/OBC. It applies the `@LMethod` annotation of the overriden method. We could say that this annotation is also a [declaration annotation](https://github.com/GlowstoneMC/Linkstone/wiki/Declaration-annotations).

`@LImplements` belongs to the interface delegation category. It has nothing to do with the declaration annotations.